src/utils/apiResponse.utils.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
PullProofAI

Consider using more descriptive names for the properties in `ControllerResponse` like `isSuccess`, `statusCode`, `message`, `responseData`, and `errorData` for better clarity.
```diff
export type ControllerResponse = {
3-   isSuccess: number;
4-   statusCode: number;
5-   message: string;
6+   responseData?: any;
7+   errorData?: any;
```


<!-- This is an auto-generated comment by PullProofAI -->